src/control/logging/syslog_test.go

```diff
@@ -66,6 +67,10 @@ func TestSyslogOutput(t *testing.T) {
 			expected: regexp.MustCompile(fmt.Sprintf(`\n[A-Za-z]{3} \d{2} \d{2}:\d{2}:\d{2} [\w\.\-]+ [^:]+: %s\n`, infoStr))},
 		"Infof": {prio: syslog.LOG_INFO, fmtFn: logger.Infof, fmtFnFmt: fmt.Sprintf("%s: %%d", infoStr), fmtFnArgs: []interface{}{42},
 			expected: regexp.MustCompile(fmt.Sprintf(`\n[A-Za-z]{3} \d{2} \d{2}:\d{2}:\d{2} [\w\.\-]+ [^:]+: %s: 42\n`, infoStr))},
+		"Notice": {prio: syslog.LOG_INFO, fn: logger.Notice, fnInput: noticeStr,
```

Comment:
Should not be syslog.LOG_NOTICE ?

Reply:
This is a cherry-pick of Kris' master commit (#9792)... The comment in syslog.go indicates that the notice-level messages should be mixed in with the info-level messages, presumably to avoid unnecessary changes in logging behavior.